bisqapps/shared/domain/src/commonMain/kotlin/network/bisq/mobile/utils/LogUtil.kt

```diff
@@ -0,0 +1,17 @@
+package network.bisq.mobile.utils
+
+import co.touchlab.kermit.Logger
+
+private val loggerCache = mutableMapOf<String, Logger>()
+
+val Any.log: Logger
```

Comment:
Instead of extending Any can we add it only to the base classes we are interested in?

Like BasePresenter , maybe BaseService? and any abstract class we are interested in. We probably don't want this dependency to affect the models for example.

Reply:
I have some service classes which have no base class... I think logging is useful in many areas at least for dev testing stuff...

Reply:
It is indeed, just don't want the dependency to every single object (that's what happens when you extend Any)

Reply:
It is only applied if log is called, no? I don't know the internals of extension functions...

Reply:
only if you use the keywords `by lazy` - but anyways your new solution is so much better :guitar: